src/services/github.js

```diff
@@ -45,14 +44,15 @@ export const fetchBaseTreeSha = async commitSha => {
   }
 }
 
-export const createTree = async (baseTreSha, fileContent) => {
+export const createTree = async (baseTreSha, chainId, fileContent) => {
```

Comment:
The GitHub service looks awesome. I notice we have an inconsistent format between the two versions of `dao-list` with regards to how we handle `token_logo` assets.